code/gradle.properties

```diff
@@ -11,19 +11,19 @@
 #
 
 org.gradle.jvmargs=-Xmx2048m
-android.injected.testOnly = false
-org.gradle.configureondemand=false
+org.gradle.caching=true
+
+android.useAndroidX=true
 
-moduleProjectName=places
 moduleName=places
-moduleAARName=places-phone-release.aar
-moduleVersion=2.1.0
+moduleVersion=3.0.0
 
 #Maven artifact
 mavenRepoName=AdobeMobilePlacesSdk
 mavenRepoDescription=Android Places Extension for Adobe Mobile Marketing
 mavenUploadDryRunFlag=false
-mavenCoreVersion=2.3.0
 
-android.disableAutomaticComponentCreation=true
-android.useAndroidX=true
+# production versions for production build
+mavenCoreVersion=3.0.0
+mavenPlacesVersion=2.0.0
```

Comment:
Why do we have `mavenPlacesVersion` here? Is this supposed to be `mavenEdgeVersion` instead?

Reply:
Removed mavenPlacesVersion. It doesn't make sense here.
And also mavenEdgeVersion is not a hard dependency on places will ignore that one too